core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy

```diff
@@ -31,12 +31,12 @@ l.layout(permission:app.SYSTEM_READ, title:my.displayName, cssclass:request.getP
                 }
 
                 div(style:"width:100%") {
-                    f.descriptorRadioList(title:_("Security Realm"), varName:"realm", instance:app.securityRealm, descriptors: h.filterDescriptors(app, SecurityRealm.all()))
+                    f.dropdownDescriptorSelector(title:_("Security Realm"), field: 'securityRealm', descriptors: h.filterDescriptors(app, SecurityRealm.all()))
                 }
             }
 
             div(style:"width:100%") {
-                f.descriptorRadioList(title:_("Authorization"), varName:"authorization", instance:app.authorizationStrategy, descriptors:h.filterDescriptors(app, AuthorizationStrategy.all()))
+                f.dropdownDescriptorSelector(title:_("Authorization"), field: 'authorizationStrategy', descriptors: h.filterDescriptors(app, AuthorizationStrategy.all()))
```

Comment:
@jglick this change breaks a CloudBees proprietary feature (security enforcement from Operations Center to controllers). Can you flag this (internally) as a breaking change please?

Reply:
reference for @cloudbees: `OperationsCenterSecurityInfo`